src/Confluent.Kafka/Consumer.cs

```diff
@@ -27,22 +27,15 @@
 
 namespace Confluent.Kafka
 {
-    /// <summary>
-    ///     Implements a high-level Apache Kafka consumer (with 
-    ///     key and value deserialization).
-    /// </summary>
-    public class Consumer<TKey, TValue> : IDisposable
+    /// <inheritdoc cref="IConsumer{TKey,TValue}"/>
```

Comment:
No, we need to keep the implementation documentation. All public code should be documented. Best thing to do for now is to use the indlude_docs.xml file